File: ts/test/exists.test.ts

import { test, describe } from 'node:test'
import { equal } from 'node:assert'


import { FuelPricesAtSpanishGasStationsSDK } from '..'


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await FuelPricesAtSpanishGasStationsSDK.test()
    equal(null !== testsdk, true)
  })

})

File: lua/test/exists_test.lua
-- FuelPricesAtSpanishGasStations SDK exists test

local sdk = require("fuel-prices-at-spanish-gas-stations_sdk")

describe("FuelPricesAtSpanishGasStationsSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
